xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStream.h

```diff
@@ -157,6 +157,14 @@ class CDVDInputStream
   virtual bool CanSeek() { return true; }
   virtual bool CanPause() { return true; }
 
+  CURL GetURL() const { return m_item.GetURL(); }
+
+  ProxyType GetProxyType() const;
```

Comment:
wrong generalization. why do you place those here. this proxy stuff is HLS and ffmpeg only